modules/apps/sync/sync-api/bnd.bnd

```diff
@@ -1,4 +1,6 @@
 Bundle-Name: Liferay Sync API
 Bundle-SymbolicName: com.liferay.sync.api
 Bundle-Version: 1.0.2
-Export-Package: com.liferay.sync.constants
\ No newline at end of file
+Export-Package: com.liferay.sync.*
```

Comment:
Do we really have to export everything? Also, please don't use wildcards but list explicitly what packages you want to export (SF enforces this).